projects/packages/my-jetpack/src/products/class-product.php

```diff
@@ -161,15 +161,6 @@ public static function get_plugin_filename() {
 		return static::$plugin_filename;
 	}
 
-	/**
-	 * This method will be called in the class initializer to perform any necessary initialization
-	 *
-	 * @return void
-	 */
-	public static function initialize() {
```

Comment:
No longer using this on any product and I have doubts we'll ever have to, so I'm removing it for now